tests/gnmi/helper.py

```diff
@@ -9,36 +10,37 @@
 GNMI_SERVER_START_WAIT_TIME = 15
 
 
-def gnmi_container(duthost):
-    global GNMI_CONTAINER_NAME
-    if len(GNMI_CONTAINER_NAME) == 0:
-        if get_image_type(duthost) == "public":
-            GNMI_CONTAINER_NAME = "telemetry"
-        else:
-            GNMI_CONTAINER_NAME = "gnmi"
-    return GNMI_CONTAINER_NAME
+@lru_cache(maxsize=None)
+class GNMIEnvironment(object):
```

Comment:
I see we have this class in both tests/gnmi and tests/telemetry, and the difference is we hardcode values vs other we pull from config_db. Is it possible to reuse same class, but pass in some input to differentiate?

Reply:
Updated.